src/java.base/share/classes/java/util/Random.java

```diff
@@ -150,6 +308,25 @@ private static long initialScramble(long seed) {
         return (seed ^ multiplier) & mask;
     }
 
+    /**
+     * Returns an instance of {@code Random} that delegates method calls to the {@link RandomGenerator}
+     * argument. If the generator is an instance of {@code Random}, it is returned. Otherwise, this method
+     * returns an instance of {@code Random} that delegates all methods except {@code setSeed} to the generator.
+     * The returned instance's {@code setSeed} method always throws {@link UnsupportedOperationException}.
+     * The returned instance is not serializable.
+     *
+     * @param generator the {@code RandomGenerator} calls are delegated to
+     * @return the delegating {@code Random} instance
+     * @throws NullPointerException if generator is null
```

Comment:
One more small thing. Add this line after the `@throws` line:

     * @since 19

Thanks.

Reply:
Added